building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
-set -eu
+set -e -u
+function cleanup {
+    rm -f resources/GIT_VERSION
+}
+trap cleanup EXIT
+cleanup
+
+set +e
+u="$(git status --porcelain)" && test -z "$u"
+GIT_DIRTY=$?
+set -e
+
+git rev-parse HEAD | tr -d '\n' >resources/GIT_VERSION
+[[ $GIT_DIRTY -ne 0 ]] && echo -n "-dirty" >>resources/GIT_VERSION
```

Comment:
You realize you can just set u above and then use test -z "$u" down here (or its negation, perhaps) instead of comparing the retval, right?

Reply:
Yes, that sounds like a better idea.